administrator/components/com_content/src/Service/HTML/Icon.php

```diff
@@ -136,6 +137,76 @@ public function edit($article, $params, $attribs = [], $legacy = false)
         return $output;
     }
 
+    /**
+     * Display an edit icon for the category.
+     *
+     * @param   object    $category  The category information.
+     * @param   Registry  $params    The item parameters.
+     * @param   array     $attribs   Optional attributes for the link.
+     * @param   boolean   $legacy    True to use legacy images, false to use icomoon based graphic.
+     *
+     * @return  string  The HTML for the category edit icon.
+     *
+     * @since   6.1.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

doing this means our build scripts will correctly set the version - it might not make it to 6.1 ;)